docs/AspDotNet.md

```diff
@@ -70,17 +70,12 @@ If you want to customize the views (adding links, etc.) you can add JavaScript f
 Settings.Current.Render.JSIncludes.Add("/Content/errors.js");
 ```
 
-If you want to store some custom key/value style data with an exception, you can set up `Settings.GetCustomData`, for example:
+If you want to store some custom key/value style data with an exception, you can use `.AddLogData` extension method, for example:
```

Comment:
This set of changes isn't right...all that stuff still works (we're using it on SO), what were you hitting problems with?

Reply:
sorry, no issue -- I was just trying to keep the different guides somewhat consistent with each other. I push an update shortly with this reverted.

Reply:
this is back to the original